pkg/apikey/apikey.go

```diff
@@ -31,13 +28,16 @@ type Key struct {
 	TkPrivateKey string `json:"-"` // do not store the private key in the metadata file
 	TkPublicKey  string `json:"public_key"`
 
-	// Underlying ECDSA keypair
-	privateKey *ecdsa.PrivateKey
-	publicKey  *ecdsa.PublicKey
-}
+	scheme signatureScheme
+
+	// Underlying ECDSA keypair (if applicable)
+	ecdsaPrivKey *ecdsa.PrivateKey
+	ecdsaPubKey  *ecdsa.PublicKey
```

Comment:
Can we define an interface which satisfies both ecdsa.PublicKey _and_ ed25519.PublicKey?

Or could we provide a get method on type `Key` which returns them?

Reply:
Done! Good suggestion.